apps/wallet-mobile/src/features/SetupWallet/useCases/RestoreWallet/RestoreWalletScreen.tsx

```diff
@@ -382,7 +380,7 @@ const useStyles = () => {
     suggestionText: {
       ...atoms.body_1_lg_regular,
       textTransform: 'none',
-      color: color.primary_c500,
+      color: color.text_primary_high,
```

Comment:
@rahulnr7 I didn't find this state in Figma, and it is using the old sub-token, please confirm with Design.